Source/Scene/Model.js

```diff
@@ -1454,6 +1454,191 @@ define([
         return getStringFromTypedArray(loadResources.getBuffer(bufferView));
     }
 
+    function shallowClone(obj) {
```

Comment:
Isn't this the same as calling [clone](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/clone.js) when `deep` is `false`, which is the default?